modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryServiceImpl.java

```diff
@@ -31,4 +33,22 @@
 )
 public class NotificationQueueEntryServiceImpl
 	extends NotificationQueueEntryServiceBaseImpl {
+
+	@Override
+	public void deleteNotificationQueueEntry(long notificationQueueEntryId)
+		throws PortalException {
+
+		notificationQueueEntryLocalService.deleteNotificationQueueEntry(
+			notificationQueueEntryId);
+	}
+
+	@Override
+	public NotificationQueueEntry resendNotificationQueueEntry(
```

Comment:
@gustavolimav can you remove method this from now? Nothing related to resend in this PR. Next one we do this.

Reply:
Ok